driver-core/src/main/java/com/datastax/driver/core/querybuilder/Clause.java

```diff
@@ -111,6 +111,78 @@ boolean containsBindMarker() {
         }
     }
 
+    static class ContainsClause extends AbstractClause {
+
+        private final List<Object> values;
+
+        ContainsClause(String name, List<Object> values) {
+            super(name);
+            this.values = values;
+
+            if (values == null)
+                throw new IllegalArgumentException("Missing values for CONTAINS clause");
+            if (values.size() > 1)
+                throw new IllegalArgumentException("Too many values for CONTAINS clause, only one value allowed");
+        }
+
+        @Override
+        void appendTo(StringBuilder sb, List<Object> variables) {
+
+
+            Utils.appendName(name, sb).append(" CONTAINS ").append("'"+values.get(0)+"'");
```

Comment:
It looks like this does cause a problem for binding prepared statements.  I added an attempt to fix this @ https://github.com/tolbertam/java-driver/commit/6d84c0c86fefa610abb5cff3c9618d5c7425e482 and added some integration tests around this.  If it looks good, you can cherry-pick it over to your branch if you like.